plugin/src/main/java/net/dzikoysk/funnyguilds/config/PluginConfiguration.java

```diff
@@ -1299,10 +1299,25 @@ public void loadProcessedProperties() {
     }
 
     public enum DataModel {
+        FLAT(null, null),
+        MYSQL("mysql", "com.mysql.cj.jdbc.Driver"),
+        MARIADB("mariadb", "org.mariadb.jdbc.Driver");
 
-        FLAT,
-        MYSQL
+        private final String jdbc;
```

Comment:
Trochę tak sobie nazwane, może jakiś databaseType czy coś? No albo na razie można po prostu robić nazwę enuma i toLowerCase, i tak wstawiać w url (może nawet bez lower case zadziała ładnie)

Reply:
Nie rozumiem cię trochę. Chodzi o to że dodatkowy enum stworzyć o nazwie DatabaseType?

Reply:
Nie, albo:
1. Zmienić nazwę tej zmiennej, żeby lepiej mówiła co to jest
2. W ogóle się pozbyć tego i po prostu używać nazwy enuma budując URL dla JDBC